### PR TITLE
Buttons put on the right side & Divider added

### PR DESCRIPTION
**Pull request by mustiwebp**
What the title says.

### CODE REVIEW

File: lib/widgets/picos_radio_select.dart
Comment (S1rDan) [inline lib/widgets/picos_radio_select.dart:1]: The divider are not quite there yet. Look at PainScale or the Mockup to see, how it should look like.

**Review by S1rDan — CHANGES_REQUESTED** [review]

File: lib/widgets/picos_radio_select.dart
Comment (mustiwebp) [reply lib/widgets/picos_radio_select.dart:1]: I added some adjustments

File: lib/widgets/picos_radio_select.dart [hunk 1]
```diff
@@ -61,7 +63,23 @@ class _PicosRadioSelectState extends State<PicosRadioSelect> {
   @override
   Widget build(BuildContext context) {
     return Column(
-      children: _createItemList(),
+      children: <ListView>[
```
Comment (S1rDan) [inline lib/widgets/picos_radio_select.dart:66]: Well two things:
1. The ListView makes the Column redundant. You should only use one or the other.
2. THe ListView makes the content scrollable even if it shouldn't scroll. Try to scroll within a PicosRadioSelect and you notice, the overscroll effect.

**Review by S1rDan — CHANGES_REQUESTED** [review]

File: lib/widgets/picos_radio_select.dart
Comment (S1rDan) [inline lib/widgets/picos_radio_select.dart:1]: There is another visual bug I noticed:

The ink splash (the effect if you hold down on an item) doesn't appear over the whole width of the parent widget anymore, like it does with SleepQuality and PainScale.

Comment (mustiwebp) [reply lib/widgets/picos_radio_select.dart:1]: Should be solved now.

File: lib/widgets/picos_radio_select.dart [hunk 2]
```diff
@@ -40,11 +40,14 @@ class PicosRadioSelect extends StatefulWidget {
 class _PicosRadioSelectState extends State<PicosRadioSelect> {
   dynamic _selectValue;
 
+  final double _distance = 15;
```
Comment (S1rDan) [inline lib/widgets/picos_radio_select.dart:43]:
```suggestion
  static const double _distance = 15;
```
One more optimization.

File: lib/widgets/picos_radio_select.dart [hunk 3]
```diff
@@ -40,11 +40,14 @@ class PicosRadioSelect extends StatefulWidget {
...
+
   List<RadioListTile<dynamic>> _createItemList() {
     return widget.selection.entries.map<RadioListTile<dynamic>>(
       (MapEntry<String, dynamic> element) {
         return RadioListTile<dynamic>(
           title: Text(element.key),
+          contentPadding: EdgeInsets.symmetric(horizontal: _distance),
```
Comment (S1rDan) [inline lib/widgets/picos_radio_select.dart:50]:
```suggestion
          contentPadding: const EdgeInsets.symmetric(horizontal: _distance),
```

File: lib/widgets/picos_radio_select.dart [hunk 4]
```diff
@@ -53,15 +56,30 @@ class _PicosRadioSelectState extends State<PicosRadioSelect> {
...
-      children: _createItemList(),
+    return ListView.separated(
+      scrollDirection: Axis.vertical,
+      shrinkWrap: true,
+      physics: const NeverScrollableScrollPhysics(),
+      itemCount: _createItemList().length,
+      itemBuilder: (BuildContext context, int index) =>
+          _createItemList()[index],
+      separatorBuilder: (BuildContext context, int index) {
+        return Divider(
```
Comment (S1rDan) [inline lib/widgets/picos_radio_select.dart:75]:
```suggestion
        return const Divider(
```

**Review by S1rDan — CHANGES_REQUESTED** [review]

**Review by S1rDan — APPROVED** [review]

**Review by S1rDan — COMMENTED** [review]
Should be fine now.

**Pull request merged**